app/models/mixins/supports_feature_mixin.rb

```diff
@@ -102,12 +102,8 @@ module SupportsFeatureMixin
     :update_network_router      => 'Network Router Update',
   }.freeze
 
-  # Whenever this mixin is included we define all features as unsupported by default.
-  # This way we can query for every feature
-  included do
-    QUERYABLE_FEATURES.keys.each do |feature|
-      supports_not(feature)
-    end
+  QUERYABLE_FEATURES.keys.each do |feature|
+    define_method("supports_#{feature}?") { false }
```

Comment:
This does only define an instance method. So we cant ask the class if it supports the feature.
```ruby
class X
  include SupportsFeatureMixin
end
X.new.supports_delete? #> false
X.supports_delete? #> undefined method
```